src/client/pages/_slug.vue

```diff
@@ -24,3 +36,43 @@ export default {
   computed: {},
 }
 </script>
+
+<style>
+@import '../components/app-core/index.css';
+
+.page__header {
```

Comment:
styling for the `page-header` can be moved to its component. then you can remove the class name `page__header`